lib/user/user-session-spec.js

```diff
@@ -0,0 +1,42 @@
+var mockMbaasServiceProxy = require('./stub/mbaas-service-proxy');
+var mediator = require('fh-wfm-mediator/lib/mediator');
+var express = require('express');
+var proxyquire = require('proxyquire');
+var sinon = require('sinon');
+var assert = require('assert');
+
+var sessionManager;
+
+describe("Mbaas Service Proxy", function() {
+
+  beforeEach(function() {
+    var verifySessionStub = mockMbaasServiceProxy.getMockVerifySessionStub();
+
+    sessionManager = proxyquire('./user-session', {
+      './mbaas-service-proxy': mockMbaasServiceProxy.getMockSessionObject(verifySessionStub)
+    });
+
+    sessionManager(mediator, express(), 'service-name-guid');
+  });
+
+  it("Testing valid session token", function(done) {
+
+    mediator.request('wfm:user:session:validate', 'myvalidsessiontoken', function(err, validationResponse) {
+      sinon.assert.calledOnce(mockMbaasServiceProxy.getMockVerifySessionStub());
```

Comment:
The

```
mockMbaasServiceProxy.getMockVerifySessionStub()
```

is creating a new stub. You want to assert on the stub that was created in the test. See above comment.